CordovaLib/CordovaLib.xcodeproj/project.pbxproj

```diff
@@ -704,25 +720,10 @@
 			};
 			name = Release;
 		};
-		1DEB921F08733DC00010E9CD /* Debug */ = {
-			isa = XCBuildConfiguration;
-			buildSettings = {
-				PUBLIC_HEADERS_FOLDER_PATH = "include/Cordova";
-				TARGETED_DEVICE_FAMILY = "1,2";
-			};
-			name = Debug;
-		};
-		1DEB922008733DC00010E9CD /* Release */ = {
-			isa = XCBuildConfiguration;
-			buildSettings = {
-				PUBLIC_HEADERS_FOLDER_PATH = "include/Cordova";
-				TARGETED_DEVICE_FAMILY = "1,2";
-			};
-			name = Release;
-		};
 		C0C01EB71E3911D50056E6CB /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				DEFINES_MODULE = YES;
```

Comment:
It's a shame that for a 1-line change (in two spots), Xcode reformats the entire project file 😞 
I'm almost tempted to just add this `DEFINES_MODULES = YES;` line with a text editor where it's needed to avoid a noisy diff

Reply:
@dpogue I think we need this noisy diff because the project file is now organized by modules.

Reply:
Also, if consumed as a submodule, this diff will show up if we don't check this in. Let's merge this PR as-is.